components/attendance/TodaySummary.tsx

```diff
@@ -122,16 +124,29 @@ const TodaySummary: StylableFC<{
             <Text type="title-medium">{t("homeroom.title")}</Text>
             {
               {
-                // If there is no homeroom content, show a button to add content.
+                // If there is no homeroom content and the user is not a
+                // student, show a button to add content.
                 empty: (
-                  <Button
-                    appearance="filled"
-                    icon={<MaterialIcon icon="add" />}
-                    onClick={() => setHomeroomView(HomeroomView.edit)}
-                    className="!justify-self-start"
-                  >
-                    {t("homeroom.action.add")}
-                  </Button>
+                  <>
+                    {mysk.user?.role === "student" && !mysk.user?.is_admin ? (
```

Comment:
The condition of whether to show the add/edit button in this component is different from the condition in the `editable` variable in `pages/classes/[classNumber]/attendance/date/[date].tsx`. This results in admin student being able to add/edit homeroom content in larger screen, while only being able to see in smaller screen. 
![image](https://github.com/user-attachments/assets/2df61877-2196-4d81-9365-6364286f44c4)
![image](https://github.com/user-attachments/assets/c4b1fb71-c04b-4ece-94dc-f9bbe415cfdd)

Reply:
I 'think' I forgot to use the editable variable, will fix that.

Reply:
Good catch! Will fix that one.